Dockerfile

```diff
@@ -5,13 +5,13 @@
 # $ docker run -d --rm -p <local-port>:80 --name malibu-container malibu
 # 
 
-FROM php:7.3-apache-buster
+FROM docker.io/php:8.2-apache-bullseye
 
 ENV JQUERY 3.6.0
 ENV CLIPBOARD 2.0.8
 
-RUN apt-get update \
-    && apt-get install --no-install-recommends -y yaz libyaz-dev wget unzip python3-bs4 python3-requests \
+RUN apt-get update && apt-get upgrade -y
```

Comment:
Is an upgrade really needed? - I think that the recommendations were more going in the direction that one should not do such an upgrade when using an official Docker image which is by itself maintained. However, now it seems that one can find pros and cons for doing this in a docker container or not.

Reply:
I removed the commit which changed that part now, because I can no longer reproduce the problem which I saw yesterday.